graylog2-server/src/main/java/org/graylog2/security/ldap/LdapSettingsImpl.java

```diff
@@ -84,6 +84,8 @@
     public static final String LDAP_GROUP_MAPPING_NAMEKEY = "group";
     public static final String LDAP_GROUP_MAPPING_ROLEKEY = "role_id";
 
+    public static final String SYSTEM_PASSWORD_PLACEHOLDER = "*************";
```

Comment:
I am a bit opposed against the idea to return a magic string to mask the password. Maybe it is a better idea to use a boolean flag (`system_password_set`) to indicate that there is an ldap system password configured, which is returned in (and only in) the DTO used by `LdapResource#getLdapSettings`? The DTOs used for setting/getting LDAP settings are already separate, so the effort should not be too big.